app/models/form_profile.rb

```diff
@@ -329,23 +329,27 @@ def vets360_contact_info_hash
     return_val
   end
 
+  def pciu_disabled?
+    Flipper.enabled?(:remove_pciu, user)
+  end
+
```

Comment:
Made pciu_disabled? to eliminate repetitive` Flipper.enabled?(:remove_pciu, user) `calls.